### DISCUSSION TITLE
How do I do inpainting with color?

### DISCUSSION THREAD

**Discussion by Njasa2k** [comfyanonymous/ComfyUI#963] [Q&A] [answered]
I can only do inpainting with a alpha mask but I want to inpaint a region with the influence of the color like I've seen in InvokeAI.

**Comment by ltdrdata** [comment]
Maybe you can extract color and convert to mask through custom nodes.

Check WAS Node Suite.

**Reply by Njasa2k** [reply]
can you show me a workflow?

**Comment by ltdrdata** [comment, marked as answer]
![color-mask](https://github.com/comfyanonymous/ComfyUI/assets/128333288/abaa99ac-5dac-4871-a572-4ef92f4af80c)

This is the example
